subtrack.DAL/Migrations/SubtrackDbContextModelSnapshot.cs

```diff
@@ -66,7 +69,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.Property<string>("Name")
                         .IsRequired()
-                        .HasMaxLength(12)
+                        .HasMaxLength(50)
```

Comment:
was this changed manually or is it just github messing around?

Reply:
auto generated during migration